src/main/resources/templates/start.html

```diff
@@ -22,34 +22,13 @@
 }
 </style>
 <body>
-<div id="one">
-<nav class="navbar navbar-expand-lg navbar-light bg-light">
-    <h2> Meal Mentor </h2>
-    <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarSupportedContent" aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation">
-        <span class="navbar-toggler-icon"></span>
-    </button>
-
-    <div class="collapse navbar-collapse" id="navbarSupportedContent">
-        <ul class="navbar-nav mr-auto">
-            <li class="nav-item active">
-                <a class="nav-link" href="#">Home <span class="sr-only">(current)</span></a>
-            </li>
-            <li class="nav-item active">
-                <a class="nav-link" href="#">Shopping List</a>
-            </li>
-        </ul>
-        <form class="form-inline my-2 my-lg-0" action ="#"  th:action="@{/searchReceipe}" th:object="${Recipe}" method="GET">
-            <input class="form-control mr-sm-2" type="search" placeholder="Search a meal ..." aria-label="Search" th:field="*{name}" >
-            <button class="btn btn-outline-info my-2 my-sm-0" type="submit">Search</button>
-        </form>
-    </div>
-</nav>
+<div th:replace="fragments/topnav :: navbar">
 </div>
-<div id="two">
```

Comment:
Just like the navBar there is no need to give this entire div an id. But if you do feel the need for one it needs to be something that tells me what I am referencing without requiring me to look at the code. 2 tells me nothing, "weeklyMeals" would probably be a better id.